mslib/mscolab/server.py

```diff
@@ -203,7 +203,7 @@ def home():
     return render_template("/index.html")
 
 
-@APP.route("/status")
+@APP.route("/status_auth")
 @conditional_decorator(auth.login_required, mscolab_settings.__dict__.get('enable_basic_http_authentication', False))
```

Comment:
I think this makes upcoming 9 consistent. It means also without /status any older version of msui knows to be outdated

We could also add a hint for that change into the 8.1.x

We could also use a complete different route name. /get_status, /info,  /status,

Reply:
Correct, I think it can stay as this. A better name would maybe be "authenticate"? but the current one is fine. Any changes required?

Reply:
We remember this with the MR from @nilupulmanodya

Reply:
Sure.. Thanks for remembering :)